htmresearch/support/neural_correlations_utils.py

```diff
@@ -61,6 +61,23 @@ def randomizeSequence(sequence, symbolsPerSequence, numColumns, sparsity, p = 0.
     numSymbolsToChange -= 1
   return randomizedSequence
 
+def highOrderSequence(sequence, symbolsPerSequence, numColumns, sparsity):
```

Comment:
The name of this function doesn't reflect what it actually does. Maybe use 'perturbSequence' or 'randomizeSequence' instead?